### DISCUSSION TITLE
Wyze RTSP with TinyCam

### DISCUSSION THREAD

**Discussion by exploitpanda** [mrlt8/docker-wyze-bridge#369] [Q&A]
Hello Github

I was finally able to get the bridge to finally start but had a bit of an issue. I was using tinycam pro to begin with but always had issues where the camera wouldn't connect cause of issues. Sometimes it would work fine. I have the RTSP all setup correctly but wasn't able to fully get it functioning on tinycam. Anyone else using tinycam and rtsp bridge together?

**Comment by mrlt8** [comment]
I believe tiny cam uses the same tutk SDK library to connect to the cameras, so you may run into the same problems as before.

Are you able to view the RTSP or HLS streams on another media player like VLC or in a browser?

**Reply by exploitpanda** [reply]
> I believe tiny cam uses the same tutk SDK library to connect to the cameras, so you may run into the same problems as before.
> 
> Are you able to view the RTSP or HLS streams on another media player like VLC or in a browser?

I was able to see the streams on RTSP VLC. What would you recommend to watch the streams? I basically want a little monitor to watch them

**Reply by mrlt8** [reply]
Hmm. The rtsp stream should work in tinycam if you can view it in VLC. Maybe a firewall issue on the machine running the bridge?

**Reply by exploitpanda** [reply]
> Hmm. The rtsp stream should work in tinycam if you can view it in VLC. Maybe a firewall issue on the machine running the bridge?

I think I do have a firewall issue but I am kinda new to this and have no clue what to do. I was getting some error like “0.0.0.0:19xx” and I found some workaround to start the container but I’m assuming that might be issue

**Reply by exploitpanda** [reply]
Well after a reboot of my computer, I can't seem to even access the RTSP in VLC

**Reply by mrlt8** [reply]
Is this with the `docker run` command from the quick start?

If so, you may need to add add the ports for the rtsp stream `-p 8554:8554`:
```
docker run  -e WYZE_EMAIL=my@email.com  -e WYZE_PASSWORD=mypassword  -p 8554:8554  -p 8888:8888 mrlt8/wyze-bridge:latest
```

**Reply by exploitpanda** [reply]
No, I created it using portainer

**Reply by mrlt8** [reply]
Can you post the error message from portainer?

**Reply by exploitpanda** [reply]
I finally got it working.

**Comment by exploitpanda** [comment]
@mrlt8 Failure
Ports are not available: listen tcp 0.0.0.0:1935: bind: An attempt was made to access a socket in a way forbidden by its access permissions. is the error I am recieving. It was working for a few days but I've got this error on Portainer when trying to start

**Reply by mrlt8** [reply]
hmm, looks to be a windows issue..? https://github.com/docker/for-win/issues/3171#issuecomment-739740248

FYI, port 1935 is only used for RTMP, so you could disable that if you're not using RTMP or select a different port..?

**Reply by exploitpanda** [reply]
> hmm, looks to be a windows issue..? [docker/for-win#3171 (comment)](https://github.com/docker/for-win/issues/3171#issuecomment-739740248)
> 
> FYI, port 1935 is only used for RTMP, so you could disable that if you're not using RTMP or select a different port..?

Hello, 

thank you so much for answering all my questions, I have tried to see if I can disable RTMP but I believe I have to have it running first. I am only using RTSP so I would like to know how to disable RTMP

**Reply by mrlt8** [reply]
Is this in portainer?
You should be able to edit the container and delete the 1935 port mapping:
<img width="958" alt="portainer 1" src="https://user-images.githubusercontent.com/67088095/168096253-3c276fd9-da8f-45e5-8f98-cbb03b0ae146.png">
